config/cesm/config_grids.xml

```diff
@@ -37,8 +37,8 @@
       <grid name="rof"       compset="DROF"        >rx1</grid>
       <grid name="rof"       compset="DROF%CPLHIST">r05</grid>
       <grid name="rof"       compset="XROF"        >r05</grid>
-      <grid name="glc"	     compset="SGLC"        >null</grid>
-      <grid name="glc"	     compset="CISM2"       >gland4</grid>
+      <grid name="glc"       compset="SGLC"        >null</grid>
+      <grid name="glc"       compset="CISM2"       >gland4</grid>
```

Comment:
This change (while a good fix of whitespace) is going to cause a conflict with the changes in #3943 . Can you please back this out here and I'll fix the whitespace in that PR?

Reply:
This PR has correct whitespace in it.  If there is a conflict the issue should be addressed in #3943.

Reply:
I have pushed a whitespace fix to #3943 . The problem is that, even with the same whitespace fix in that PR, merging these two branches will produce a merge conflict. Since resolving merge conflicts can be error-prone, I was asking if we can preemptively solve the problem by backing out the whitespace change here and just letting it be fixed when #3943 is merged. If you give me your okay, I'm happy to make that change myself. The end result will be the same, but the merge conflict will be avoided.

Reply:
sure, go ahead

Reply:
Done; thanks